hw/bitstream/vivado/BUILD

```diff
@@ -156,7 +156,33 @@ filegroup(
     tags = ["manual"],
 )
 
+# CW340 bitstream
+fusesoc_build(
+    name = "fpga_cw340",
+    testonly = True,
+    srcs = [
+        "//hw:all_files",
+        _FPGA_CW310_TESTROM,
```

Comment:
Eventually, the CW340 likely will be a separate target altogether, but for now, the CW310 and CW340 are compatible.

Reply:
Should we rename it to `_FPGA_CW3xx_TESTROM`?

Reply:
No, I think that would be premature. From my POV, there is no assumption of sharing software builds with CW310, and this is merely a placeholder until we decide what to do. If anything, perhaps I should create an `_FPGA_CW340_TESTROM` that just happens to point to the same target as `_FPGA_CW310_TESTROM`.